Runtime/Core/SubdivideOnVertex.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+
+
+namespace UnityEngine.ProBuilder
+{
+    [DisallowMultipleComponent]
+    [RequireComponent(typeof(ProBuilderMesh))]
+    [ExcludeFromPreset, ExcludeFromObjectFactory]
+    public sealed class SubdivideOnVertex : MonoBehaviour
```

Comment:
let's talk about this approach, not sure about using a monobehaviour here

Reply:
Oops, this was only a test functionality I did when starting in the team, I'm removing this class